go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	github.com/moby/sys/signal v0.7.0
 	github.com/moby/sys/symlink v0.2.0
 	github.com/moby/sys/user v0.1.0
+	github.com/opencontainers/distribution-spec/specs-go v0.0.0-20231012181449-3f685837a664
```

Comment:
Why not import 1.0.1?

Also: if we only need a single type, it might be easier to redefine that type rather than importing the whole library.

Reply:
This one import the whole library is better, it includes all of the distribution's request and response type definitions.
I will change the version later.

Reply:
Do we need any of those other types?  We're just using `ErrorResponse` here.

Reply:
For now, no. Your mean copy the `ErrorResponse` definition here. Both are ok.

Reply:
Done. Copied here. ^_^